spec/index.html

```diff
@@ -2437,7 +2437,27 @@ <h2 id="media-type-registration">Media Type Registration</h2>
             <dt>Required parameters:</dt>
             <dd>N/A</dd>
             <dt>Optional parameters:</dt>
-            <dd>N/A</dd>
+            <dd>
+              <dl>
+                <dt>charset</dt>
+                <dd>
+                  The <code>charset</code> parameter may be provided to definitively
+                  specify the document's character encoding,
+                  overriding any character encoding declarations in the document.
+                  The parameter's value must be one of the labels of the character
+                  encoding used to serialize the file.
+                </dd>
+                <dt>projection</dt>
+                <dd>
+                  The <code>projection</code> parameter may be provided to definitively
+                  specify the document's associated Tiled Coordinate Reference System (TCRS),
+                  overriding any projection declarations in the document.
```

Comment:
<q>overriding any projection declarations in the document</q> refers to taking precedence over e.g. `<meta http-equiv="projection" ...>` and `<meta name="projection" ...>`. Which I'm not sure is/should be true.

Reply:
I agree. Let's remove it.

Reply:
Okay, I'll remove the addition of the `projection` parameter entirely per https://github.com/Maps4HTML/MapML/pull/171#discussion_r622254869.